airflow/models.py

```diff
@@ -403,6 +406,10 @@ def collect_dags(
                                 filepath, only_if_updated=only_if_updated)
                     except Exception as e:
                         logging.warning(e)
+        Stats.gauge(
+            'collect_dags', (datetime.now() - start_dttm).total_seconds(), 1)
```

Comment:
dag_collection_time_seconds? Also, I'm not too familiar with how these metrics appear on the user end. Is it just this string or is there any additional context about which module it came from?